src/encoder.rs

```diff
@@ -199,6 +262,37 @@ impl<W: Write> Encoder<W> {
     pub fn set_adaptive_filter(&mut self, adaptive_filter: AdaptiveFilterType) {
         self.adaptive_filter = adaptive_filter;
     }
+
+    /// Sets the dispose operation for each frame.
```

Comment:
Copy paste error?

Reply:
Yup, but there is also a some documentation missing so I won't worry much about it right now. Once we agree on an implementation I would fill in those gaps